saml/gatein-saml-plugin/src/main/java/org/gatein/sso/saml/plugin/valve/AbstractSPFormAuthenticator.java

```diff
@@ -382,6 +382,13 @@ private boolean isGlobalLogout(Request request) {
      * @return true if this is a local SAML logout
      */
     private boolean isLocalLogout(Request request) {
+        try {
+            if (request.getCharacterEncoding()==null) {
```

Comment:
I am not sure we need to check as we have to force the encoding to UTF-8, can we just set its value to UTF-8 ?

Reply:
In fact, if the request have already an encoding, I dont want to enforce it. It should keep the existing one